src/jarabe/view/buddyicon.py

```diff
@@ -109,7 +117,7 @@ def __get_battery(self):
     def __status_tooltip(self, has_battery=False):
         disk_usage = (self.used * 100) / self.total
         battery = ''
-        if has_battery:
+        if self.has_battery:
```

Comment:
Your change here to use `self.has_battery` means the keyword argument is not needed?